docs/integrations/pytorch.rst

```diff
@@ -3,109 +3,370 @@
 Neptune-PyTorch Integration
 ===========================
 
-Neptune has implemented an integration with the PyTorch machine learning library.
+.. note::
+
+    Neptune integrates with both pure PyTorch and many libraries from the PyTorch Ecosystem.
+    You may want to check out the following integrations:
+
+    - :ref:`PyTorch Lightning <integrations-pytorch-lightning>`
+    - :ref:`Fastai and Fastai2 <integrations-fastai>`
+    - :ref:`PyTorch Ignite <integrations-pytorch-ignite>`
+    - :ref:`Catalyst <integrations-catalyst>`
+    - :ref:`Skorch <integrations-skorch>`
+
+    For pure PyTorch integration, read on.
+
+|colab-script-neptune|
+
+What will you get with this integration?
+----------------------------------------
+
+|pytorch-tour-loom|
+
+|Pytorch| is an open source machine learning framework commonly used for building deep neural network models.
```

Comment:
machine learning -> maybe deep learning

Reply:
took it from their site actually, but I think you are right -> it is in deep learning frameworks in our docs right?